Makefile

```diff
@@ -32,6 +32,8 @@ help:
 	@echo "        Create Debian repository fixtures with missing Packages files for some architectures."
 	@echo "    fixtures/debian-flat"
 	@echo "        Create Debian repository fixtures in flat repository format."
+	@echo "    fixtures/debian11"
```

Comment:
Should we perhaps call this `debian-mixed` instead of debian11, since we refer to this repo format as "mixed" from within the pulp_deb code? In principle this format could be used by repos other than debian11 (even though debian11 is the main example I am aware of).